app/decorators/container_node_decorator.rb

```diff
@@ -0,0 +1,5 @@
+class ContainerNodeDecorator < MiqDecorator
+  def self.listicon_image
+    ActionController::Base.helpers.image_path("100/container_node.png")
```

Comment:
Please keep this in line with the rest.. `listicon_image` never calls `image_path`, only returns the string.

It's the caller's responsibility to call `image_path`.